cmake/onnxruntime_providers.cmake

```diff
@@ -768,19 +768,46 @@ if (onnxruntime_USE_TENSORRT)
 endif()
 
 if (onnxruntime_USE_VITISAI)
-  file(GLOB_RECURSE onnxruntime_providers_vitisai_cc_srcs CONFIGURE_DEPENDS
-    "${ONNXRUNTIME_ROOT}/core/providers/vitisai/*.h"
+  if ("${GIT_COMMIT_ID}" STREQUAL "")
+  execute_process(
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    COMMAND git rev-parse HEAD
```

Comment:
why is GIT_COMMIT_ID needed specifically for vitisai?

Reply:
Used to obtain the version of ONNXRuntime Vitis AI EP in the runtime phase, which is very useful for reproducing issues in the troubleshooting  phase.